src/commands/completion/completion.ts

```diff
@@ -66,7 +66,7 @@ export const completionGenerate = async (options: OptionValues, command: BaseCom
       },
     ])
     if (compinitAdded) {
-      await fs.readFile(zshConfigFilepath, 'utf8', (err, data) => {
+      fs.readFile(zshConfigFilepath, 'utf8', (_err, data) => {
```

Comment:
sorry for the unrelated changes, just fixing some minor warnings as I encounter them